index.html

```diff
@@ -3857,6 +3857,73 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media type used by this specification relies on JSON-LD, JSON-LD
+processing is not required to be performed, since JSON-LD is JSON. Some
+scenarios where processing a <a>verifiable credential</a> or a <a>verifiable
+presentation</a> as JSON is desirable include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+Prior to verification when using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation, as described in Section
+<a href="#data-schemas"></a>.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into systems that store or index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed for securing mechanisms that require an understanding of
+and processing of JSON-LD.
+            </li>
+            <li>
+When an application chooses to process the media type using the `+json`
+structured media type suffix.
+            </li>
+          </ul>
+
+          <p>
+That is, pure JSON processing is allowed as long as the document being
+consumed or produced is a <a>conforming document</a>. If pure JSON processing
+is desired, an implementer is advised to follow the following rule:
+          </p>
+
+          <ul>
+            <li>
+Ensure that all values associated with a `@context` property are in the
+expected order, the contents of the context files match known good
+cryptographic hashes for each file, and the contents are known to be valid.
+            </li>
+          </ul>
+
+          <p>
+Using static context files with a JSON Schema is one acceptable approach to
+implementing the rule above. This can ensure proper term identification,
+typing, and order, when a JSON document is processed as JSON-LD.
+          </p>
+
+          <p>
+The rule above guarantees semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism. While JSON-LD processors will use the
+specific mechanism provided and can verify that all terms are correctly
+specified, JSON-based processors implicitly accept the same set of terms without
+testing that they are correct. In other words, the context in which the data
```

Comment:
We should clarify that this testing doesn't happen at runtime. The assumption is that someone, at some point, has ensured that the contexts (with well-known hash values) are already valid (this someone could be the same issuer that a verifier trusts, for example). Then anyone can confidently write interoperable software that does not need to perform JSON-LD transformations at runtime.

```suggestion
specified, JSON-based processors implicitly accept the same set of terms without
testing that they are correct at runtime. In other words, the context in which the data
```

Reply:
Fixed in 02fa6bf19378478aca22ba05fc705fece35b4076.